Task1/Program.cs

```diff
@@ -10,7 +10,556 @@ class Program
+                    break;
+                case "лето":
+                    Console.WriteLine("лето - summer");
+                    break;
+                case "осень":
+                    Console.WriteLine("осень - autumn");
+                    break;
+                case "снег":
+                    Console.WriteLine("снег - snow");
+                    break;
+                case "дождь":
+                    Console.WriteLine("дождь - rain");
+                    break;
+                case "солнце":
+                    Console.WriteLine("солнце - sun");
+                    break;
+                case "тучи":
+                    Console.WriteLine("тучи - clouds");
+                    break;
+                case "ветер":
+                    Console.WriteLine("ветер - wind");
+                    break;
+                case "гроза":
+                    Console.WriteLine("гроза - thunderstorm");
+                    break;
+
+                default:
+                    Console.WriteLine("you shoud enter excisting word!");
+                    break;
+            }
+        }
+
+        public static void Task8()
+        {
+            Console.WriteLine("Enter long service: ");
+            double longService = Convert.ToDouble(Console.ReadLine());
+            Console.WriteLine("Enter salary: ");
+            int salary = Convert.ToInt32(Console.ReadLine());
+            if (longService > 0 && longService < 5)
+            {
+                double premium = salary * 0.1;
+                Console.WriteLine($"Salary is: {salary} long service is: {longService} premium is: {premium} total: {salary + premium}");
+            }
+            if (longService > 4 && longService < 10)
+            {
+                double premium = salary * 0.15;
+                Console.WriteLine($"Salary is: {salary} long service is: {longService} premium is: {premium} total: {salary + premium}");
+            }
+            if (longService > 9 && longService < 16)
+            {
+                double premium = salary * 0.25;
+                Console.WriteLine($"Salary is: {salary} long service is: {longService} premium is: {premium} total: {salary + premium}");
+            }
+            if (longService > 15 && longService < 21)
+            {
+                double premium = salary * 0.35;
+                Console.WriteLine($"Salary is: {salary} long service is: {longService} premium is: {premium} total: {salary + premium}");
+            }
+            if (longService > 20 && longService < 26)
+            {
+                double premium = salary * 0.45;
+                Console.WriteLine($"Salary is: {salary} long service is: {longService} premium is: {premium} total: {salary + premium}");
+            }
+            if (longService > 25)
+            {
+                double premium = salary * 0.5;
+                Console.WriteLine($"Salary is: {salary} long service is: {longService} premium is: {premium} total: {salary + premium}");
```

Comment:
I think you've noticed that this Console.WriteLine is invoked several times.
Also the string is absolutely the same.
So we can move it to the end of the method and this code will be used for all conditions.